frontend/src/core/config/config.ts

```diff
@@ -27,9 +27,13 @@ export const UserConfigSchema = z
         auto_instantiate: z.boolean(),
       })
       .default({ auto_instantiate: true }),
+    display: z
+      .object({
+        theme: z.enum(["light", "dark"]).default("light"),
```

Comment:
its an enum to support "system" or other themes in future. but we expose it as a checkbox to the user